src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_OF_ELEMENTS = 10;
```

Comment:
This variable`s name could use a minor change
```suggestion
    private static final int MAX_ELEMENTS = 10;
```